instrumentation/net/http/otelhttp/transport.go

```diff
@@ -121,27 +174,46 @@ func (t *Transport) RoundTrip(r *http.Request) (*http.Response, error) {
 		return res, err
 	}
 
+	// metrics
+	metricAttrs := append(labeler.Get(), semconvutil.HTTPClientRequestMetrics(r)...)
+	if res.StatusCode > 0 {
+		metricAttrs = append(metricAttrs, semconv.HTTPStatusCode(res.StatusCode))
+	}
+	o := metric.WithAttributes(metricAttrs...)
+	t.requestBytesCounter.Add(ctx, bw.read, o)
+	// For handling response bytes we leverage a callback when the client reads the http response
+	readRecordFunc := func(n int64) {
+		t.responseBytesCounter.Add(ctx, n, o)
+	}
+
+	// traces
 	span.SetAttributes(semconvutil.HTTPClientResponse(res)...)
 	span.SetStatus(semconvutil.HTTPClientStatus(res.StatusCode))
-	res.Body = newWrappedBody(span, res.Body)
+
+	res.Body = newWrappedBody(span, readRecordFunc, res.Body)
+
+	// Use floating point division here for higher precision (instead of Millisecond method).
+	elapsedTime := float64(time.Since(requestStartTime)) / float64(time.Millisecond)
```

Comment:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4590

Reply:
I agree with the general desire for this codebase to follow the OTEL specification, but I also see some value in following existing convention and then having a more thought out approach to migrating towards seconds across the entire codebase. 

This is the same reason I stuck with the incorrectly named metrics, so that this code would help reach feature parity with service side metrics

Reply:
Thank you. I'll leave this up to maintainers to decide.

Reply:
It should be `ms` like defiened in https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/metrics/semantic_conventions/http-metrics.md#http-client. Resolving.